akd/src/append_only_zks.rs

```diff
@@ -634,6 +635,13 @@ impl Azks {
             })
             .collect();
 
+        if let Some(labels) = marker_labels {
+            lookup_nodes.extend(labels.iter().map(|l| AzksElement {
+                label: *l,
+                value: AzksValue(EMPTY_DIGEST),
+            }))
+        }
+
```

Comment:
nit: I feel like there should be a way to consolidate this so that we don't repeat the ```
.map(|l| AzksElement {
                label: *l,
                value: AzksValue(EMPTY_DIGEST),
            }))
            ```
 segment of code twice

Reply:
I had issues with references when I tried to extend the `lookup_infos` iterator after the flat-map. Perhaps there is a valid syntax to correctly reference them

Reply:
Let's create an issue out of this so that we remember to come back to it and also add a comment in the code

Reply:
https://github.com/facebook/akd/issues/437